src/components/SideBar/SideBarComponents/FacetFilters.js

```diff
@@ -95,13 +93,7 @@ const FacetPanel = ({ classes }) => {
     if ((filter[1].length >= 1) && (document.getElementById(`filterGroup_${filter[0]}`))) {
       const filterLabel = facetSearchData.filter((word) => word.datafield === filter[0]);
       document.getElementById(`filterGroup_${filter[0]}`).innerHTML = `${filterLabel[0].label}`;
-      if (filterLabel[0].section === 'Filter By Cases') {
-        document.getElementById(`filterGroup_${filter[0]}`).style.color = '#10A075';
-      } else if (filterLabel[0].section === 'Filter By Samples') {
-        document.getElementById(`filterGroup_${filter[0]}`).style.color = '#10BEFF';
-      } else if (filterLabel[0].section === 'Filter By Files') {
-        document.getElementById(`filterGroup_${filter[0]}`).style.color = '#E636E4';
-      }
+      document.getElementById(`filterGroup_${filter[0]}`).style.color = facetSectionVariables[filterLabel[0].section].color;
     } else if (document.getElementById(`filterGroup_${filter[0]}`)) {
       const filterLabel = facetSearchData.filter((word) => word.datafield === filter[0]);
       document.getElementById(`filterGroup_${filter[0]}`).innerHTML = `${filterLabel[0].label}`;
```

Comment:
Can you use React.useRef() instead of pure js as Karan mentioned

Reply:
Okay, I will try